geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/IngestCommand.scala

```diff
@@ -200,4 +200,10 @@ trait IngestParams extends OptionalTypeNameParam with OptionalFeatureSpecParam w
 
   @Parameter(names = Array("--src-list"), description = "Input files are text files with lists of files, one per line, to ingest.")
   var srcList: Boolean = false
+
+  @Parameter(names = Array("--no-job-status"), description = "Progress of the ingest job is not printed to the console, however, the application will close when job completes.")
+  var noJobStatusTracking: Boolean = false
+
+  @Parameter(names = Array("--no-wait"), description = "This application closes when ingest job is submitted. Useful for launching jobs with a script.")
+  var noWaitForCompletion: Boolean = false
```

Comment:
can we rename this to `--no-tracking` and update the description to "Return once the job is submitted, instead of printing job status updates to the console"

Reply:
We need to update the docs for these commands as well

Reply:
Added